os/include/tinyara/streams.h

```diff
@@ -229,9 +298,37 @@ extern "C" {
  *
  ****************************************************************************/
 
+/**
+ * @ingroup STREAMS_LIBC
+ * @brief Initializes a stream for use with a fixed-size memory buffer
+ * @param[in] User allocated, uninitialized instance of struct lib_meminstream_s
```

Comment:
This function has three arguments, param[in], but there is only one [in] in doxygen comments . Is it right?